CMakeLists.txt

```diff
@@ -74,6 +75,9 @@ if(NOT MSVC AND NOT ANDROID)
         -pedantic
         -Wextra
     )
+    if (FACELIFT_ENABLE_GCOV)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -g -O0 -fprofile-arcs -ftest-coverage -fPIC")
```

Comment:
-Wall is set above
-g -O0 are set by CMake when setting CMAKE_BUILD_TYPE=Debug
why -fPIC ?
Use check_supported_compiler_flags to set compiler flags

Reply:
Q: why -fPIC?
A: Info from official docs:
"Additionally, we compile with the -fprofile-arcs -ftest-coverage -fPIC compiler options, which add logic to generate output files that can be processed by the gcov command."
See more: https://gcovr.com/en/stable/guide.html#tabular-output-of-code-coverage
My understanding:
The reason of using -fPIC when build '.so' is to avoided that the same code will be used of many processes that mapped the same physical memory to various virtual addresses.

Perhaps using -fPIC is not for our case.